main.go

```diff
@@ -22,6 +22,16 @@ import (
 // OutputFormatVersion will allow us to filter output files downstream.
 const OutputFormatVersion = '1'
 
+// AvailableStats records which statistics are available on this platform, and
+// so shouldn't cause an error if they're empty.
+type AvailableStats struct {
+	CPUPercent bool
```

Comment:
Note to self: I might need to bake `MemoryPercent` too if https://github.com/mozilla-releng/resource-monitor/pull/18 lands before this.